### ISSUE TITLE
Screen Reader - About Items

### ISSUE THREAD

**Issue by EvanSchallerer** [osu-cass/SampleItemsWebsite#124]
Cannot access Item Types description with screen reader.

**Comment by alexlepinski** [comment]
fixed
